include/openthread.h

```diff
@@ -1802,7 +1673,7 @@ OTAPI void OTCALL otPlatformReset(otInstance *aInstance);
  *
  * @param[in]  aInstance A pointer to an OpenThread instance.
  */
-void otFactoryReset(otInstance *aInstance);
+OTAPI void OTCALL otFactoryReset(otInstance *aInstance);
```

Comment:
`openthread.h` is being used both as the wrapper and the actual implementation here. The goal here is to provide the same interface so that code that already exists calling these control APIs is easily portable to Windows.